strawberry/experimental/pydantic/object_type.py

```diff
@@ -73,8 +86,36 @@ def get_type_for_field(field: ModelField):
     return type_
 
 
+if TYPE_CHECKING:
+    PydanticModel = TypeVar("PydanticModel", bound=BaseModel)
+
+    class StrawberryTypeFromPydantic(Protocol[PydanticModel]):
+        """This class does not exist in runtime.
+        It only makes the methods below visible for IDEs"""
```

Comment:
could we move this definition to a new file and import it instead? 😊

Reply:
> pyright is able to find to_pydantic on the strawberry type, but that's because it doesn't use the mypy plugin :)

seems like pyright implements class decorators out of the box. [mypy still lags behind](https://github.com/python/mypy/issues/3135). booo.

Reply:
@patrick91 moved to a new file and edited RELEASE.md. Hopefully its clearer now!